hosting.md

```diff
@@ -99,6 +99,8 @@ title: Hosting a Review Club Meeting
 
 #### Making a New Post
 
+Clone the website repository [located on GitHub](https://github.com/bitcoin-core-review-club/website).
```

Comment:
To avoid frustration with lacking permissions to push to this repo, would suggest calling it forking instead of cloning?
```suggestion
[Fork](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks) the website repository [located on GitHub](https://github.com/bitcoin-core-review-club/website).
```